app/Models/User.php

```diff
@@ -71,6 +71,14 @@ public function linkedSocialAccounts()
         return $this->hasMany(LinkedSocialAccount::class);
     }
 
+    /**
+     * Get the announcements createad by the user
```

Comment:
Correct typo "createad" to "created"
Add a return type

    /**
     * Get the announcements created by the user.
     *
     * @return \Illuminate\Database\Eloquent\Relations\HasMany
     */